src/graphnet/models/model.py

```diff
@@ -146,7 +165,7 @@ def predict(
     def predict_as_dataframe(
         self,
         dataloader: DataLoader,
-        prediction_columns: List[str],
+        prediction_columns: List[str] = None,
```

Comment:
If we want to make it optional whether to specify a list.
```suggestion
        prediction_columns: Optional[List[str]] = None,
```